src/templates/Resources/PractitionerRole.hbs

```diff
@@ -4,31 +4,40 @@
         "resourceType": "PractitionerRole",
         "id":"{{ID}}",
         {{>DataType/XCNPractitionerRole.hbs XCN=PD1-4}},
-        {{>DataType/XCNPractitionerRole.hbs XCN=PRT-5}},

```

Comment:
I think that PRT-5 does have an identifier component?

Reply:
this line still exists on right hand side (moved a little lower)

Reply:
Sorry! I caught a couple of these but not all of them.